addons/event/tests/test_event_mail_schedule.py

```diff
@@ -22,7 +22,7 @@ def test_event_mail_schedule(self):
 
         # freeze some datetimes, and ensure more than 1D+1H before event starts
         # to ease time-based scheduler check
-        now = datetime(2021, 3, 20, 14, 30, 15)
+        now = datetime(2021, 3, 20, 14, 30, 15, 123456)
```

Comment:
Hm.. maybe add a comment to say why we add microseconds? Like `# microseconds should not matter`
Because I'm sure I'm going to have forgotten this change next week ^^

Reply:
Please take a seat
nboulif made a clearer comment, should be up anytime!